Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -9,8 +8,7 @@
 from typing import Dict, List, Optional, Tuple, Iterable
 import urllib3
 
-urllib3.disable_warnings()
-
+requests.packages.urllib3.disable_warnings()

```

Comment:
I don't know, it was here before me.

Reply:
First of all, this is part of their standard - https://xsoar.pan.dev/docs/integrations/code-conventions
Second of all, it is used of disabling insecure warnings from the requests library.  
InsecureRequestWarning is a warning that occurs when a request is made without certificate verification. In Python, this warning happens for requests sent from requests and urllib libraries.

Since we do not force certificate verification demisto's client code, this error might show up.